### PR TITLE
Adds randomly picking titlescreens and a new titlescreen

### PR DESCRIPTION
**Pull request by Niknakflak**
This compiles but I didn't test it to see if it worked.  This is to truly add in those "titlescreens" that admins replace (mostly PAS) and to randomly pick from said titlescreens available.  

I only added one so far and it's not complete, maybe if kor's GF can do something so we don't have to have the black on both sides, but Im also going to see if I can animate some of it's colors like our original titlescreen has as well as add "/tg/station" somewhere?  Not sure yet.

Here's a picture 
image

Comment and subscribe and tell me what's broken because I did this during my two hour gap between classes. V(._.)V

### CODE REVIEW

File: code/game/turfs/closed.dm [hunk 1]
```diff
@@ -20,19 +20,20 @@
 
 
 
-
 /turf/closed/indestructible/splashscreen
 	name = "Space Station 13"
 	icon = 'icons/misc/fullscreen.dmi'
-	icon_state = "title"
+	icon_state = null
```
Comment (RemieRichards) [inline code/game/turfs/closed.dm]: ```title_1``` please, for the sake of the map.

**Comment by KorPhaeron** [comment]
Gonna have to talk to her (and maybe shaps?) and consider if shes okay with her art being used under the /tg/ license. Will be a couple hours before I have the chance

**Comment by KorPhaeron** [comment]
Should probably hold off doing more work till then

**Comment by PKPenguin321** [comment]
what do the dimensions of title cards have to be

File: code/game/turfs/closed.dm (hunk 1 above)
Comment (Niknakflak) [inline code/game/turfs/closed.dm]: I wasn't aware it could have a default, I thought it had to be null, will change soon.

**Comment by Niknakflak** [comment]
@KorPhaeron 
ok

@PKPenguin321 
480x480

**Comment by KorPhaeron** [comment]
Also I would rather have one title screen for each map so I dont keep accidentally joining maps I dislike

**Comment by Niknakflak** [comment]
That would be such a limiting feature

**Comment by Niknakflak** [comment]
Look at the status tab like a normal person

**Comment by Incoming5643** [comment]
A neat little idea but

1. We need more than just the one alternate screen
2. Implying a roundtype on said screen is kinda bad (unless most/all roundtypes have a screen)

Personally I always wanted to make it so every map had their own title screen with the differently shaped stations in the same style as the traditional one too.

**Comment by Niknakflak** [comment]
1. feel free to contribute 
2. why, because it'll imply a roundtype? Well people will be pleasantly surprised when "surprise" it's not nuke op.  

That last thing is possible, the feature exists for it to be possible but nobody cares to do it.

**Comment by KorPhaeron** [comment]
She is not comfortable with the current asset license for /tg/

**Comment by Niknakflak** [comment]
:c

**Comment by KorPhaeron** [comment]
I don't know why we have some dumb "people can sell our stuff" license anyway

**Comment by optimumtact** [comment]
You can license the artwork under another license, it the cc by sa is just the default

**Comment by optimumtact** [comment]
If you do license it under another license the sprite needs to go in it's own folder and have a license.txt file with it

**Comment by optimumtact** [comment]
never mind

**Comment by ike709** [comment]
rip

**Pull request closed without merging**